starwars/src/components/CharacterTemplate.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Character from './Character'
+
+function CharacterTemplate(props) {
+    return (
+        props.starwarsChars.map(function (eachChar) {
+            return <Character eachChar={eachChar}/>
```

Comment:
Can be refactored to:
`props.starwarsChars.map(eachChar => <Character eachChar={eachChar}/>)`
for a more modern implementation